services/nodeping/nodeping-status.service.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+const Joi = require('joi')
+
+const rowSchema = Joi.object().keys({ su: Joi.boolean() })
+
+const schema = Joi.array()
+  .items(rowSchema)
+  .min(1)
+
+/*
+ * this is the checkUuid for the NodePing.com (as used on the [example page](https://nodeping.com/reporting.html#results))
+ */
+const sampleCheckUuid = 'jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei'
+
+class NodePingStatus extends BaseJsonService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'Status',
+    }
+  }
+
+  static get route() {
+    return {
+      base: 'nodeping/status',
+      pattern: ':checkUuid',
+      queryParams: [
+        'label',
```

Comment:
No need to re-define `label` as a query param here, it's already available for all badges